content/cspm/aws/acm/acm-certificate-validation.md

```diff
@@ -12,7 +12,7 @@ breadcrumb_remediation_child_name: ACM
 ---
 ### Quick Info
 
-| | |
+| Issue ID | AZU001 |
```

Comment:
There isn't any ID associated here. Were you trying to use this as a key for something? If so use The actual Plugin Title has it is unique.